### DISCUSSION TITLE
Prune unreachable objects

### DISCUSSION THREAD

**Discussion by hinell** [community/community#77839] [Repositories]
### Select Topic Area

General

### Body

I would really like to be able to prune my repo from unused objects without recreating entire repo.

**Comment by dakshithadissanayaka** [comment]
Sure, here is a summary of the discussion and a possible solution:

**Issue:**

A user is trying to set up a GitHub Pages site for a URL that they own but are having trouble verifying ownership. The user has added a CNAME record to their DNS settings, but GitHub Pages is not recognizing it.

**Possible Solution:**

The user can try the following steps to resolve the issue:

1. **Double-check CNAME record:** Ensure that the CNAME record is configured correctly. The CNAME record should point to the subdomain of your custom domain that you want to use for your GitHub Pages site, and it should point to `cname.pages.github.com`.

2. **Wait for DNS propagation:** Allow time for the DNS propagation to complete. DNS propagation can take up to 48 hours, so it may take some time for the CNAME record to be recognized by GitHub Pages.

3. **Check for conflicting records:** Verify that there are no conflicting DNS records for the subdomain you're trying to use. Check for any A records or other CNAME records that might be interfering with the verification process.

4. **Clear cache:** Try clearing your browser's cache and cookies to ensure that you are not viewing an outdated version of the page.

5. **Contact GitHub support:** If the issue persists after trying these steps, contact GitHub support for further assistance. They can provide more specific troubleshooting steps and check for any underlying issues with your domain or DNS configuration.

By following these steps, the user can increase their chances of successfully verifying domain ownership and setting up their GitHub Pages site.
